packages/vc-verification/src/resolver/Ipfs-credential-resolver.ts

```diff
@@ -225,4 +248,23 @@ export class IpfsCredentialResolver implements CredentialResolver {
       )
     ).filter(isVerifiableCredential);
   }
+
+  /**
+   * Fetches DID Document for the given DID
+   * @param did subject DID
+   * @param didDocumentCache Cache to store DIDDocument. Cache is updated with Document retrieved for the DID
+   * @returns
+   */
+  async getDIDDocument(
```

Comment:
I think `getDIDDocument` could be private and then it doesn't need to be in the interface. Does this sound right?
```suggestion
  async private getDIDDocument(
```

Reply:
@jrhender We need this public, since it is used by `ClaimIssuerVerification` - https://github.com/energywebfoundation/ew-credentials/blob/853c4135777f19df46edea6de2b0d43ebe76d05e/packages/vc-verification/src/verifier/claim-issuer-verification.ts#L120

Reply:
And this would also allow PDA to pass cacheClient call inside this to fetch from SSI-Hub

Reply:
https://github.com/energywebfoundation/ew-credentials/blob/853c4135777f19df46edea6de2b0d43ebe76d05e/packages/vc-verification/src/verifier/claim-issuer-verification.ts#L120

At this line, why not use `this._issuerDefResolver` to get the `issuerDIDDoc`?
It is counterintuitive to me that the class that the `ClaimIssuerVerification` class has an `isserDefResolver` but doesn't use it to get the `issuerDIDDoc`.

Reply:
@jrhender I didn't get the correlation between the RoleDefinition resolver and DID Document. To me DID Document is more related to holder's credential. `this._issuerDefResolver` only resolves `RoleDefintion` from ENS.

Reply:
Oh you're right! A moment of confusion on my part 😅